scripts/variantstore/wdl/GvsAssignIds.wdl

```diff
@@ -168,3 +173,49 @@ task AssignIds {
   }
 }
 
+task CreateCostObservabilityTable {
+  input {
+    String project_id
+    String dataset_name
+  }
+
+  String cost_observability_json =
+                                 '[ { "name": "call_set_identifier", "type": "STRING", "mode": "REQUIRED" }, '     + # The name by which we refer to the callset
+                                 '  { "name": "step", "type": "STRING", "mode": "REQUIRED" }, '                    + # The name of the core GVS workflow to which this belongs
+                                 '  { "name": "call", "type": "STRING", "mode": "NULLABLE" }, '                    + # The WDL call to which this belongs
+                                 '  { "name": "shard_identifier", "type": "STRING", "mode": "NULLABLE" }, '        + # A unique identifier for this shard, may or may not be its index
+                                 '  { "name": "call_start_timestamp", "type": "TIMESTAMP", "mode": "REQUIRED" }, ' + # When the call logging this event was started
+                                 '  { "name": "event_timestamp", "type": "TIMESTAMP", "mode": "REQUIRED" }, '      + # When the observability event was logged
+                                 '  { "name": "event_key", "type": "STRING", "mode": "REQUIRED" }, '               + # The type of observability event being logged
+                                 '  { "name": "event_bytes", "type": "INTEGER", "mode": "REQUIRED" } ] '             # Number of bytes reported for this observability event
+
+  meta {
+    # not volatile: true, always run this when asked
+  }
+  command <<<
+    set -o xtrace
+
+    echo "project_id = ~{project_id}" > ~/.bigqueryrc
+    TABLE="~{dataset_name}.cost_observability"
+
+    # Check that the table has not been created yet
+    set +o errexit
+    bq show --project_id ~{project_id} $TABLE > /dev/null
+    BQ_SHOW_RC=$?
+    set -o errexit
+
+    if [ $BQ_SHOW_RC -ne 0 ]; then
+      PARTITION_STRING="--time_partitioning_field call_start_timestamp --time_partitioning_type DAY"
+      echo "making table $TABLE"
+      echo '~{cost_observability_json}' > schema.json
+      bq --location=US mk ${PARTITION_STRING} --project_id=~{project_id} $TABLE schema.json
+    fi
+  >>>
+  runtime {
+    docker: "us.gcr.io/broad-dsde-methods/variantstore:ah_var_store_2022_05_16"
```

Comment:
why are we using our custom docker?

I'm worried about this getting stale (though I suppose it wont matter for this use case)

I feel like we tend to use:     docker: "us.gcr.io/broad-gatk/gatk:4.1.7.0"